bin/setup

```diff
@@ -24,6 +24,7 @@ FileUtils.chdir APP_ROOT do
 
   puts "\n== Preparing database =="
   system! "bin/rails db:prepare"
+  system! "RAILS_ENV=test bin/rails db:setup"
```

Comment:
@danielburnley just checking this is the best way to dothis

Reply:
I always forget `bin/setup` exists - seems like a sensible thing to do yea 👍🏻